### ISSUE TITLE
After installed from app store, can't found app in extensions menu

### ISSUE THREAD

**Issue by ober01** [quicktype/quicktype-xcode#9]
![image](https://user-images.githubusercontent.com/5435782/34931198-7a24b62a-f9de-11e7-91bc-a349bd51db46.png)

**Comment by dvdsgl** [comment]
Strange. I’ll look into it.

You can try one of our developer builds: https://install.appcenter.ms/orgs/quicktype/apps/quicktype-xcode/distribution_groups/Xcode%20Testers

It’s unsigned so you’ll need to right-click to open it.

**Comment by dvdsgl** [comment]
What Xcode version do you have?

**Comment by ober01** [comment]
Version 9.2

**Comment by ober01** [comment]
I tried your developer build, the same result

**Comment by ober01** [comment]
I also use Xvim2 plugin https://github.com/XVimProject/XVim2, 
And for working this plugin need signing Xcode https://github.com/XVimProject/XVim2/blob/master/SIGNING_Xcode.md
Maybe this the reason

**Comment by dvdsgl** [comment]
The App Store version is signed... Any chance you could try on another computer? I’ll try on a new Mac tomorrow.

**Comment by dvdsgl** [comment]
I know this sounds silly, but have you relaunched System Preferences? It needs to reload to discover the extension, apparently.

**Comment by ober01** [comment]
I removed Xvim and XcodeSigner, but it doesn't help.

**Comment by mikealex** [comment]
I have similar issues. After installing from the Mac App Store, it does appear in my Xcode extensions and has been enabled, but the menu option doesn't appear in Xcode.

I downloaded the latest upgrade today, but still no luck.

<img width="780" alt="screen shot 2018-01-21 at 12 37 31 pm" src="https://user-images.githubusercontent.com/938573/35197139-625558c2-fea9-11e7-8fd7-8f24464f7914.png">
<img width="279" alt="screen shot 2018-01-21 at 12 40 05 pm" src="https://user-images.githubusercontent.com/938573/35197140-626c9154-fea9-11e7-9f7e-627e738c8949.png">
<img width="535" alt="screen shot 2018-01-21 at 12 44 43 pm" src="https://user-images.githubusercontent.com/938573/35197141-627df8ea-fea9-11e7-9f51-401bdddc2f19.png">
<img width="698" alt="screen shot 2018-01-21 at 12 44 58 pm" src="https://user-images.githubusercontent.com/938573/35197142-628eca44-fea9-11e7-83e9-064fe52d8a10.png">
<img width="377" alt="screen shot 2018-01-21 at 12 52 46 pm" src="https://user-images.githubusercontent.com/938573/35197190-11851cba-feaa-11e7-878d-e0efba760f56.png">

**Comment by dvdsgl** [comment]
Alright I’ll try to contact Apple about it.

**Comment by dvdsgl** [comment]
Would either of you be willing to have a screen sharing session where we try to build and debug the extension on your machine together?

**Comment by dvdsgl** [comment]
Does running `sudo /usr/libexec/xpccachectl` and rebooting your Mac work? After some research, this is a commonly suggested remedy.

**Comment by mikealex** [comment]
Google confirmed, xpccachectl no longer exists in High Sierra.

I'm around until 5pm EST if you'd like to do a screen share.

**Comment by mikealex** [comment]
BTW, brought down the git version. Build fails because it's missing quicktype.js

**Comment by dvdsgl** [comment]
Follow the instructions in the README to build it.

**Comment by mikealex** [comment]
Still no dice. I'm not a Node developer, so I may not have all the tools installed.

Mikes-rMBP:quicktype-xcode mikealex$ npm install

> quicktype-xcode@1.0.0 install /Users/mikealex/X-Code/quicktype-xcode
> npm run build


> quicktype-xcode@1.0.0 build /Users/mikealex/X-Code/quicktype-xcode
> script/build.sh

xcode-select: error: tool 'agvtool' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
xcode-select: error: tool 'agvtool' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! quicktype-xcode@1.0.0 build: `script/build.sh`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the quicktype-xcode@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mikealex/.npm/_logs/2018-01-21T20_15_21_930Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! quicktype-xcode@1.0.0 install: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the quicktype-xcode@1.0.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mikealex/.npm/_logs/2018-01-21T20_15_21_959Z-debug.log
Mikes-rMBP:quicktype-xcode mikealex$

**Comment by dvdsgl** [comment]
Interesting. Do you have more than one Xcode installed? Try:

```
xcode-select --install # Install Command Line Tools if you haven't already.
sudo xcode-select --switch /Library/Developer/CommandLineTools # Enable command line tools
```

**Comment by mikealex** [comment]
Same result

**Comment by mikealex** [comment]
Only one Xcode installed

**Comment by dvdsgl** [comment]
Did the command still generate quicktype.js? It's only failing when it tries to update the Xcode project version, which is not a necessary step anyway.

**Comment by mikealex** [comment]
Yes, and now it builds in Xcode.

**Comment by dvdsgl** [comment]
Make sure you choose the quicktype-xcode target when running form Xcode, this will launch a debug instance of Xcode with a black icon.

**Comment by mikealex** [comment]
It ran, but still not appearing in Editor menu.

**Comment by dvdsgl** [comment]
So strange. Anythjng useful in the debug output?

**Comment by mikealex** [comment]
Ok, it's weird. If I open an existing Swift project, the menu option isn't there. I created a new Swift project, and when I open that project the menu options do appear. This happens in both the debug Xcode and the normal Xcode.

**Comment by dvdsgl** [comment]
Maybe there is some versioning data in the Xcode project file that determines whether it allows extensions to load? That is very weird.

**Comment by dvdsgl** [comment]
@ober01 does it work if you create a new project?

**Comment by ober01** [comment]
no, in my case it doesn't appear in extensions. Look first comment

**Comment by jigar007** [comment]
Same thing happening with me, can't find "XCode Source Editor" option under Extensions

**Comment by yaalisri** [comment]
Does it support Xcode 12? I am using Big Sur and QuickType is not showing as an Xcode extension. Please refer to this [link](https://developer.apple.com/forums/thread/669335). Any update on whether this will be fixed anytime soon?

**Comment by dvdsgl** [comment]
Sorry, we are not actively working on this project. PRs are welcome to update this, though.
